pkgs/os-specific/linux/criu/default.nix

```diff
@@ -22,8 +24,9 @@ stdenv.mkDerivation rec {
   ];
 
   enableParallelBuilding = true;
+  depsBuildBuild = [ protobufc buildPackages.stdenv.cc libpaper asciidoc xmlto ];
   nativeBuildInputs = [ pkg-config docbook_xsl which makeWrapper docbook_xml_dtd_45 python3 python3.pkgs.wrapPython perl ];
```

Comment:
asciidoc and xmlto should be in nativeBuildInputs

libpaper should be in nativeBuildInputs with a comment that its for paperconf 

whats buildPackages.stdenv.cc needed for

Reply:
> asciidoc and xmlto should be in nativeBuildInputs
>
> libpaper should be in nativeBuildInputs with a comment that its for paperconf

Right. I think I've figured out the various dependency specifications for cross compilation now. This was the result of some experimentation which apparently happened to work (presumably because what these tools generate is arch-independent by definition).

> whats buildPackages.stdenv.cc needed for

Without knowing too much about the specific build process of `criu` itself, it does invoke `HOSTCC` which fails if an appropriate compiler is not present. I assume it uses it for code generation during build? This is added as per the Nixpkgs manual for when a derivation needs to build binaries for the host platform.